runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/utils.kt

```diff
@@ -16,4 +16,17 @@ fun Project.isAndroidProject() = try {
     false
 }
 
+fun Project.isNotMultiplatformProject() = !isMultiplatformProject()
```

Comment:
Functions like this should be internal IMO!

Reply:
Agreed, fixed